code/src/nuvla/api/api.py

```diff
@@ -508,7 +506,7 @@ def delete(self, resource_id) -> CimiResponse:
 
         """
         resource = self.get(resource_id=resource_id)
-        operation_href = self._cimi_find_operation_href(resource, 'delete')
+        operation_href = resource.id
         return CimiResponse(self._cimi_delete(resource_id=operation_href))
```

Comment:
IMO, having `operation_href` var is unnecessary. I suggest directly providing `resource.id` to the `resource_id` kwarg.